src/agent/src/confidential_data_hub/image.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2021 Alibaba Cloud
+// Copyright (c) 2021, 2023 IBM Corporation
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use safe_path::scoped_join;
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+
+use anyhow::{anyhow, bail, Context, Result};
+use kata_sys_util::validate::verify_id;
+use oci_spec::runtime as oci;
+
+use crate::rpc::CONTAINER_BASE;
+
+use kata_types::mount::KATA_VIRTUAL_VOLUME_IMAGE_GUEST_PULL;
+use protocols::agent::Storage;
+
+pub const IMAGE_WORK_DIR: &str = "/run/image-rs/";
```

Comment:
Do you think that changing this image working directory from `/run/kata-containers/image/` is necessary? I'm conscious that there might be an image to docs/implementations with this as I believe some people were investigating pre-caching OCI images into their guest images for performance reasons with very large models and this might break them?

Reply:
Go back to the original one, thanks